File: CloudDelivery/CloudDelivery.Data/Entities/SalePoint.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace CloudDelivery.Data.Entities
{
    [Table("SalePoints")]
    public class SalePoint : BaseModel<int>
    {
        [ForeignKey("User")]
        public int? UserId { get; set; }

        public virtual User User { get; set; }

        public string City { get; set; }

        public string Address { get; set; }

        public string LatLng { get; set; }

        public string Marker { get; set; }
    }
}

File: CloudDelivery/CloudDelivery/app/Models/Salepoints/Salepoint.ts
import { GeoPosition } from "../GeoPosition";

export class SalePoint {
    constructor() {
        this.LatLng = new GeoPosition();
    }
    UserId: number;
    City: string;
    Address: string;
    LatLng: GeoPosition;
    Marker: string;
}